docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -8,6 +8,14 @@
 
 MTA version 6.2.0 has the following issues.
 
+.Re-enabling Keycloak breaks MTA
+
+Keycloak is enabled by default. After re-enabling Keycloak, you cannot perform any actions in the MTA web console after logging in again.
+
```

Comment:
It looks good, I'd like to suggest one minor change though.

Keycloak is enabled by default. If you disable and then re-enable Keycloak, you cannot perform any actions in the MTA web console after logging in again.

Reply:
Changed. Thanks